awx/main/validators.py

```diff
@@ -197,3 +198,12 @@ def vars_validate_or_raise(vars_str):
         return vars_str
     except ParseError as e:
         raise RestValidationError(str(e))
+
+
+def validate_system_path(value):
+    """
+    Validate that string passed as a path actually exists in the
+    file system. If not, raise a ValidationError.
+    """
+    if not os.path.exists(value):
+        raise ValidationError(_('{} is not a valid path choice').format(value))
```

Comment:
Do we want to do some more checks here that the selected path isn't obviously terrible (and/or "immediately blocked by bwrap")?

Reply:
Eh, I'm not sure we need this.  We already have specific code that explicitly whitelists the virtualenv for the current job run (cc @chrismeyersfsu):

https://github.com/ansible/awx/blob/devel/awx/main/tasks.py#L1001

Reply:
OK. More concerned about someone entering "CUSTOM_VIRTUALENV_PATH=/root" or someplace else they shouldn't.

Reply:
Meh.  Do dumb things, get dumb prizes.